docs/proposals/1374-multi-cluster-inference/README.md

```diff
@@ -0,0 +1,622 @@
+- InferencePool is meant to be a replacement for a Service so it may seem counterintuitive for a user to create a Service to achieve multi-cluster inference.
+
+## Option 2: New MCS API
+
+One of the key pain points we’re seeing here is that the current iteration of the MCS API requires a tight coupling between name/namespace and kind, with Service being the only kind of backend supported right now. This goes against the broader SIG-Network direction of introducing more focused kinds of backends (like InferencePool). To address this, we could create a resource that has an `exportRef` that allows for exporting different types of resources.
+
+While we were at it, we could combine the separate `export` and `import` resources that exist today, with `export` acting as the (optional) spec of this new resource, and `import` acting as `status` of the resource. Instead of `import` resources being automatically created, users would create them wherever they wanted to reference or export something to a MultiClusterService.
+
+Here’s a very rough example:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: MultiClusterService
+metadata:
+  name: bookinfo
+  namespace: bookinfo
+spec:
+  exportRef:
+    group: v1
+    kind: Service
+    name: bookinfo
+    scope: ClusterSet
+status:
+  conditions:
+  - type: Accepted
+    status: "True"
+    message: "MultiClusterService has been accepted"
+    lastTransitionTime: "2025-03-30T01:33:51Z"
+  targetCount: 1
+  ports:
+  - protocol: TCP
+    appProtocol: HTTP
+    port: 8080
+```
+
+### Open Questions
+
+#### Export Semantics
+
+- Finalize annotation key/values and ClusterSet discovery. Should we introduce a first-class exportPolicy field later?
+- Should we allow per-cluster export configuration (weights, region, SLO tags) at export time?
+
+#### InferencePool status surface
+
+- Do we extend InferencePool.status to publish EPP Service type/addresses/ports, health, metrics, and optional secret refs to simplify imports?
+- Should EPP Deployment/Pod discovery be standardized (labels/port names) for health/metrics auto-discovery?
+
+#### Security
+
+- Standard way to bootstrap mTLS between importing IG and exported EPP/parents, e.g. use BackendTLSPolicy?
+- Should the MCI controller mirror secrets into the importing cluster (and how to scope/rotate them)?
+- Do we need a ReferenceGrant-like mechanism for cross-namespace secrets referenced by InferencePoolImport?
+
+#### Scheduling and Policy
+
+- Do we define a common ReachabilityMode enum (Endpoints vs Parents) as above? Is a mixed strategy allowed per cluster?
+- Should we define a standard cluster preference knob (e.g., PreferLocal, Any, region-affinity, weights) on InferencePoolImport.status or IG-local policy CRD?
+
+#### Topology and Reachability
+
+- For Parents mode, do we require “HTTPRoute sameness” or any guarantees between exporting/importing clusters?
```

Comment:
What I would recommend is that its a two-tier approach:
* The front gateway does the routing, weight selection, etc. It then picks a specific InferencePool in a specific cluster
* The "Parent" must serve that InferencePool. So doing its own routing or weight selecting or etc on that would be _maybe possible_ but a bad pattern.
Rather than trying to do the same routing in both places